regolith/schemas.py

```diff
@@ -183,23 +183,26 @@
                      'aka': ['Columbia University', 'Columbia'],
                      'city': 'New York',
                      'country': 'USA',
-                     'departments': {
-                         'physics': 'Department of Physics',
-                         'chemistry': 'Department of Chemistry',
-                         'apam': 'Department of Applied Physics'
-                                 'and Applied Mathematics',
-                         'cheme': 'Department of Chemical'
-                                  'Engineering',
-                         'cs': 'Computer Science Department',
-                         'eaee': 'Department of Earth and'
-                                 'Environmental Engineering'
-                     },
+                     'departments': {{'_id': 'physics',
```

Comment:
I think you have an extra curly brace here. That denotes a set syntax which is causing problems.